apps/shared/y4m.c

```diff
@@ -492,6 +493,27 @@ avifBool y4mWrite(const char * outputFilename, const avifImage * avif)
         fprintf(stderr, "WARNING: writing alpha is currently only supported in 8bpc YUV444, ignoring alpha channel: %s\n", outputFilename);
     }
 
+    if (avif->transformFlags & AVIF_TRANSFORM_CLAP) {
+        avifCropRect cropRect;
+        avifDiagnostics diag;
+        if (avifCropRectConvertCleanApertureBox(&cropRect, &avif->clap, avif->width, avif->height, avif->yuvFormat, &diag) &&
+            (cropRect.x != 0 || cropRect.y != 0 || cropRect.width != avif->width || cropRect.height != avif->height)) {
+            // TODO: Implement, see https://github.com/AOMediaCodec/libavif/issues/2427
+            fprintf(stderr,
+                    "Warning: Clean Aperture values were ignored, the output image was NOT cropped to rectangle {%u,%u,%u,%u}\n",
+                    cropRect.x,
+                    cropRect.y,
+                    cropRect.width,
+                    cropRect.height);
+        }
+    }
+    if (avifImageGetExifOrientationFromIrotImir(avif) != 1) {
+        // TODO: Rotate the samples.
```

Comment:
Nit: Follow the Style Guide's recommendation on [TODO comments](https://google.github.io/styleguide/cppguide.html#TODO_Comments). Add a user name or bug number.